dissect/target/loaders/tar.py

```diff
@@ -34,14 +34,94 @@ def __init__(self, path: Path | str, **kwargs):
                 "Consider uncompressing the archive before passing the tar file to Dissect."
             )
 
-        self.tar = tarfile.open(fileobj=path.open("rb"))
+        from dissect.target.loaders.containerimage import ContainerImageTarSubLoader
+
+        self.SUB_LOADERS = [
+            ContainerImageTarSubLoader,
+            GenericTarSubLoader,  # should be last
+        ]
```

Comment:
Maybe move this to a 
```python
__subloaders__ = [
    ContainerImageTarSubLoader,
    GenericTarSubLoader,
]
```
In the class? Can probably use `import_lazy` to avoid the circular import.